cmd/upgrade.go

```diff
@@ -67,6 +69,11 @@ func (d *diffCmd) isAllowUnreleased() bool {
 	return d.allowUnreleased || d.install
 }
 
+func (d *diffCmd) isRemoteAccessAllowed() bool {
```

Comment:
adding some unit tests will be better.

Reply:
@yxxhero Are you saying we should have a unit test for checking all the combinations of variables involved in L73?

Reply:
yeah.